models/models.articles.js

```diff
@@ -34,3 +34,20 @@ exports.selectArticleById = (id) => {
       return rows;
     });
 };
+
+exports.selectCommentsByArticleId = (id) => {
+  return db
+    .query(
+      `SELECT comment_id, comments.votes, comments.created_at, comments.author, comments.body FROM articles JOIN comments ON articles.article_id = comments.article_id WHERE articles.article_id=$1 ORDER BY created_at DESC;`,
+      [id]
+    )
+    .then(({ rows, rowCount }) => {
+      if (rowCount === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: "Path not found",
```

Comment:
we don't need this `if ( rowCount === 0) return Promise.reject` section - some articles will have no comments and that is okay - we don't want to throw errors for that 

You'll have to check that the `article_id` is in the database another way  ( you can make use of another model you made on a previous ticket `selectArticleByID` which is already set to throw an error if the `article_id` is not found )